client/src/components/modals_and_popovers/StatelessModal.tsx

```diff
@@ -6,7 +6,7 @@ import { Modal } from "react-bootstrap";
 import { trivial_text_maker } from "src/models/text";
 import "./bootstrap_modal_exstension.scss";
 
-interface StatelessModalProps {
+type StatelessModalProps = typeof StatelessModal.defaultProps & {
```

Comment:
Looks to have the same `any` issue from [this](https://github.com/TBS-EACPD/infobase/pull/1191#pullrequestreview-710647649) comment.
![image](https://user-images.githubusercontent.com/11301919/126349661-6703d90e-11ad-463c-8219-40d623831d8b.png)

Reply:
And oops, this and other files are being updated on both this and Ivan's branch. Guess I could have asked you to pause this branch until his was done.

Reply:
I'll drop the commits on this file